rewrite-java/src/main/java/org/openrewrite/java/ChangeMethodName.java

```diff
@@ -27,6 +28,9 @@
 @EqualsAndHashCode(callSuper = false)
 public class ChangeMethodName extends Recipe {
 
+    //A valid java method starts with a letter, an underscore or a $
```

Comment:
```suggestion
    /** A valid java method starts with a letter, an underscore or a $ **/
```
This is valid information valuable enough to include in the Java-Doc. While having it in the source code is helpful, that alone may not cover all needs.